design-components/packages/railz-components-angular/package.json

```diff
@@ -28,7 +28,7 @@
   "peerDependencies": {
     "@angular/common": "~13.2.2",
     "@angular/core": "~13.2.2",
-    "@railzai/railz-uikit": "^0.0.0"
+    "@railzai/railz-uikit": "^0.0.1-alpha.1"
```

Comment:
This could be:
"@railzai/railz-uikit": "latest"

That way we don't need to update this every release. This was auto-generated?

Reply:
yeah this was automatic - not sure where it came from